_posts/2021-04-17-mapping-tutorial.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title: CargoRocket Mapathon Anleitung
+author: Henri Chilla
+excerpt: Eine Anleitung zum erfolgreichen Mappen für den CargoRocket Mapathon.
+description: Tutorial zum CargoRocket Mapathon
+---
+
+Um an unserem Mapathon teilzunehmen, ist nicht viel notwendig! Hier gehen wir kurz darauf ein, wie an dem Datensammeln mitgewirkt werden kann.
+
+## OpenStreetMap
+Für einen guten Index und ein gutes Routing sind gute Daten notwendig. Hierfür beziehen wir unsere Daten von [OpenStreetMap](https://www.openstreetmap.org) (OSM). An diesem Open-Source Projekt kann sich nach dem Wikipedia-Prinzip jede Person beteiligen und beliebige Daten eintragen. Dementsprechend ist es möglich, wichtige Daten für Lastenräder zu erfassen. Diese nützen dann aber auch allen anderen Nutzer:innen, die auf die Daten zugreifen, bspw. auch Rollstuhlfahrenden.
+
+## Einführung OSM
+
+*Wenn du bereits bei OSM aktiv bist, kannst du diesen Abschnitt vermutlich überspringen.*
+
+Für das Eintragen der Daten wird zunächst ein OSM-Nutzerkonto benötigt, welcher [hier](https://www.openstreetmap.org/user/new) erstellt werden kann. Für das Bearbeiten existieren verschiedenste Editoren. Für den Einstieg sind folgende am besten geeignet: [ID-Editor](https://learnosm.org/de/beginner/id-editor/) und [StreetComplete](https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete). Für unterwegs auf dem Smartphone existieren auch [Vespucci](https://play.google.com/store/apps/details?id=de.blau.android&hl=de&gl=US) (Android) und [Go Map!!](https://apps.apple.com/de/app/go-map/id592990211) (iOS) um eine Auswahl zu erwähnen.
+
+Eine allgemeine Einführung gibt es bei [LearnOSM](https://learnosm.org/de/), denn dies soll keine Anleitung für OSM selbst sein.
+
+## Tags
+
+Wir haben verschiedene Faktoren identifiziert, welche wichtig für Lastenrad-GIS-Applikationen sind. Daraus haben wir dann die entsprechenden OSM-Tags abgeleitet. Für unsere Auswertungen verwenden wir folgende Tags, welche die Eigenschaften von Wegen und Barrieren beschreiben:
+
+* Radwege
+  *  [`surface=*`](https://wiki.openstreetmap.org/wiki/DE:Key:surface)
+  *  [`smoothness=*`](https://wiki.openstreetmap.org/wiki/DE:Key:smoothness)
+  *  [`width=*`](https://wiki.openstreetmap.org/wiki/DE:Key:width)
+*  Poller
+   *  [`maxwidth:physical=*`](https://wiki.openstreetmap.org/wiki/DE:Key:maxwidth:physical)
+
```

Comment:
Zusätzlich sollten noch nicht in OSM getaggte Barrieren wie Poller, Umlaufgitter und Bordsteine hinzugefügt werden:
- 'barrier' = bollard, cycle_barrier, kerb

Reply:
Diese Liste von Tags sollte nur Eigenschaften auflisten, die für uns relevant sind nicht die "Haupt"-Tags selbst. Oder würdest du diese auch hinzufügen?

Reply:
ich hatte das so verstanden, dass hier alle relevanten Tags, die im Mapathon erfasst werden sollen gelistet sind. 
Wir haben jetzt ja zusätzlich zu den ergänzenden Tags noch aufgenommen, dass wir zählen, wieviel neue Barrieren / Bordsteine erfasst werden.
Unten kommen die Tags ja "nur noch" unter 'Beispiele', wo beschrieben ist, wie genau etwas getaggt werden soll.
Insofern würde das für mich schon Sinn machen das hier noch zu erwähnen.

Reply:
hab gerade gesehen, dass du die Subtags für Bordsteine noch hinzugefügt hast. Dann ist das vielleicht auch verständlich genug so.
Oder einfach noch in Klammern dahinter?
`Poller (barrier=bollard)` & `Umlaufgitter (barrier=cycle_barrier`)